src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -48,23 +53,29 @@
 class OboGraphToFHIRConverter(DataModelConverter):
     """Converts from OboGraph to FHIR."""
 
-    def dump(self, source: GraphDocument, target: str = None) -> None:
+    # TODO: `retention` is temporary. I think we likely have to give this a different name/functionality, as well as put
+    #  it in the CLI and possibly elsewhere. I'm also doing this a pure way, passing it to dump and then further down,
+    #  rather than an instance property. - Joe 2022/12/01
+    def dump(self, source: GraphDocument, target: str = None, retention: str = ['maximal', 'minimal'][0]) -> None:
```

Comment:
I also got `src/oaklib/converters/obo_graph_to_fhir_converter.py:59:14: BLK100 Black would make changes.`
I can't see what it recommends, but it's probably `[0]`. I can just default the string and add the valid params to the docstring, but I like to do this in a more formal way outside of comments and more in something like a type system, but I don't actually know how you do /if you can do that in Python (tell it which param values are valid).

Reply:
I spoke w/ Harshad and I ran `black` manually